imagery/imageryintro.html

```diff
@@ -209,11 +209,13 @@ <h3>Image classification</h3>
   <li> Supervised classification (<a href="i.gensigset.html">i.gensigset</a>,
    <a href="i.smap.html">i.smap</a>)</li>
   </ul>
+  </li>
```

Comment:
"Radiometric classification:" doesn't close it's li inside the top ul. I double checked it with indented formatting to find what was missing,

![image](https://github.com/user-attachments/assets/99349c85-bbdd-494f-8e11-ecf97823d046)

So, total 3 added lines are needed, we've got 2: 
![image](https://github.com/user-attachments/assets/0f296d3c-e900-4b3a-9029-18eaf71e117c)

Reply:
There's also a typo in that file I think in "atmosphericic" ->"atmospheric"?

Reply:
Fixed in 0e9e3849b33a538c4621021b3896df9da9604fda

Reply:
The page now passes https://validator.w3.org/